src/user/models/user.entity.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToMany,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { DonationEntity } from '../../donation/models/donation.entity';
+import { SocialMediaPostEntity } from '../../social-media-post/models/social-media-post.entity';
+import { ImageEntity } from '../../image/models/image.entity';
+import { FeatCompletionEntity } from '../../feat/models/feat-completion.entity';
+
+@Entity('users')
+export class UserEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true })
+  email: string;
+
+  @Column({ unique: true })
+  username: string;
+
+  @Column()
+  password: string;
```

Comment:
If we're thinking of using something like Auth0, then this will probably be redundant, but let's keep it that way for the time being.

Reply:
We will still keep the hash of the password in the db, won't we?